packages/fasd/build.sh

```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/clvv/fasd
+TERMUX_PKG_DESCRIPTION="To fastly access files and folders"
+TERMUX_PKG_VERSION=1.0.1
+TERMUX_PKG_SHA256=88efdfbbed8df408699a14fa6c567450bf86480f5ff3dde42d0b3e1dee731f65
+TERMUX_PKG_SRCURL=https://github.com/clvv/fasd/archive/${TERMUX_PKG_VERSION}.tar.gz 
+TERMUX_PKG_MAINTAINER="Sam @SamSanuch"
+TERMUX_PKG_DEPENDS=
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS=
```

Comment:
`TERMUX_PKG_DEPENDS` and `TERMUX_PKG_EXTRA_CONFIGURE_ARGS` aren't mandatory, so they can be deleted